### ISSUE TITLE
--compiler option fails with relative path

### ISSUE THREAD

**Issue by niho** [rtfeldman/node-test-runner#212]
```
$ ./node_modules/.bin/elm-test --compiler ./node_modules/.bin/elm-make 
/bin/sh: ../node_modules/.bin/elm-package: No such file or directory
Unhandled exception while running the tests: { Error: Command failed: ./node_modules/.bin/elm-package install --yes
/bin/sh: ../node_modules/.bin/elm-package: No such file or directory
...
```

**Comment by rtfeldman** [comment]
@niho Thanks! Can you post a [SSCCE](http://sscce.org/) of this?

**Comment by niho** [comment]
It is trivial to reproduce and the example I gave above should be enough to get the error. Assuming you have elm and elm-test installed using npm in ./node_modules:

    $ ./node_modules/.bin/elm-test --compiler ./node_modules/.bin/elm-make

The above command should work according to the documentation, but does not because the relative path to elm-make does not get expanded to an absolute path. elm-test seems to change the current working dir when it runs the tests, which causes the problem (the relative path is not valid inside the ./tests directory).

To me it seems silly and annoying to require the --compile option at all. It should obviously just find elm-make and elm-package in ./node_modules/.bin/ automatically. The reasoning behind removing that feature does not seem very solid to me.

**Comment by rtfeldman** [comment]
Fixed in 0.18.9!
